code/modules/mob/new_player/new_player.dm

```diff
@@ -150,6 +168,22 @@
 			to_chat(usr, "\red The round is either not ready, or has already finished...")
 			return
 
+		// Warn the player if they are trying to spawn without a brain
+		var/datum/body_modification/mod = client.prefs.get_modification(BP_BRAIN)
+		if(istype(mod, /datum/body_modification/limb/amputation))
+			if(alert(src,"Are you sure you wish to spawn without a brain? This will likely cause you do die immediately. \
+			              If not, go into the Augmentation section of Setup Character and check you do not have Removed for the brain category.", \
```

Comment:
```suggestion
			              If not, go into the Augmentation section of Setup Character and check that you do not have Removed for the brain category.", \
```

Reply:
Engrish and not enough idiot proofing

should be:

"If not, go to the **Augmentation** section of **Setup Character** and change the "brain" slot from _Removed_ to "brain"*"

*or whatever is there, I never felt the urge to remove my brain